### ISSUE TITLE
Game has bad stuttering when playing full screen

### ISSUE THREAD

**Issue by pseudohuman55** [Mr-Wiseguy/Zelda64Recomp#175]
## If you have a crash on startup, please make sure your graphics drivers are up to date before submitting a bug report.

**Describe the bug**
Game has bad stuttering when playing full screen
This didn't occur on first launch

**To Reproduce**
1. Launch in full screen mode 
2. Make any input after game launches

**Expected behavior**
no stuttering.

**Screenshots**
Please attach a screenshot of the bug.

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Version: 23H2
 - CPU: Intel Core I5 12400f
 - GPU: Radeon RX 6700xt-
![image_2024-05-17_162336336](https://github.com/Mr-Wiseguy/Zelda64Recomp/assets/170142138/367eb64f-311c-465c-ab72-4c8963ee4a8c)
  Driver Version: 23.40.33.01-240508a1-402593C-AMD-Software-Adrenalin-Edition

**Additional context**
Adrenalin shows really hard micro stuttering taking FPS from 165 to 55 tops. Didn't occur at first launch but only after. Closed Adrenalin upon reading known issues but didn't fix. Playing in windowed mode fixes the issue but can't go back into full screen or issue comes back.  Issue also only comes up after making an input after game MM has launched

**Comment by Amiiban** [comment]
Same problem, Radeon RX 6800 and Ryzen 7 5700X cpu. Maybe it has to do with freesync? or AMD gpus in general?

**Comment by Kikatzu** [comment]
I have the same issue. It is probably caused by the Vsync in your graphics card driver settings.
In my case I disabled the setting in the NVIDIA Control Panel and the framerate hit a stable 60 FPS.
Unfortunately, this slows down the UI and causes heavy screen tearing. Not really a fix, but an explanation.

**Comment by Amiiban** [comment]
don't have nvidia control panel on amd but a similar option, using freesync monitors screen tearing isn't a big problem - still a big issue though

**Comment by Triticum0** [comment]
Is this a dupe of #75

**Comment by Triticum0** [comment]
@DarioSamo can this be closed?

**Comment by Reonu** [comment]
Closing as duplicate of #75
